go/cmd/vtctldclient/command/backups.go

```diff
@@ -28,11 +29,56 @@ import (
 	vtctldatapb "vitess.io/vitess/go/vt/proto/vtctldata"
 )
 
-// GetBackups makes a GetBackups gRPC call to a vtctld.
-var GetBackups = &cobra.Command{
-	Use:  "GetBackups <keyspace/shard>",
-	Args: cobra.ExactArgs(1),
-	RunE: commandGetBackups,
+var (
+	// Backup makes a Backup gRPC call to a vtctld.
+	Backup = &cobra.Command{
+		Use:                   "Backup [--concurrency <concurrency>] [--allow-primary] <tablet_alias>",
+		Short:                 "Uses the BackupStorage service on the given tablet to create and store a new backup.",
+		DisableFlagsInUseLine: true,
+		Args:                  cobra.ExactArgs(1),
+		RunE:                  commandBackup,
+	}
+	// GetBackups makes a GetBackups gRPC call to a vtctld.
+	GetBackups = &cobra.Command{
+		Use:  "GetBackups <keyspace/shard>",
+		Args: cobra.ExactArgs(1),
+		RunE: commandGetBackups,
+	}
+)
+
+var backupOptions = struct {
```

Comment:
nit: this could be added to the var block above

Reply:
I'm trying to match the structure of the other command files, which go like:

```go
var (
    allCommandsGoHere
)

var firstCommandOptions

func firstCommand

func secondCommandWithNoOptions

var thirdCommandOptions

func thirdCommand

func init
```

Reply:
for example: https://github.com/vitessio/vitess/blob/main/go/cmd/vtctldclient/command/cells.go